unit_tests/tests/test_etb.cpp

```diff
@@ -90,3 +90,23 @@ TEST(etb, testTargetTpsIsFloatBug945) {
 	engine->etbControllers[0]->PeriodicTask();
 	ASSERT_NEAR(51.6605, engine->engineState.targetFromTable, EPS4D);
 }
+
+TEST(etb, etbTpsSensor) {
+	// Throw some distinct values on the TPS sensors so we can identify that we're getting the correct one
+	Sensor::setMockValue(SensorType::Tps1, 25.0f);
+	Sensor::setMockValue(SensorType::Tps2, 75.0f);
+
+	// Test first throttle
+	{
+		EtbController etb;
+		etb.init(nullptr, 0, nullptr);
+		EXPECT_EQ(etb.observePlant().Value, 25.0f);
```

Comment:
This is the kind of unit testing I like being able to do - testing one thing at a time, no extra dependencies you have to set up (like injecting an engine reference or something)